lib/datadog/tracing/configuration/settings.rb

```diff
@@ -445,6 +445,34 @@ def self.extended(base)
                   )
                 end
               end
+
+              # Key-value map for explicitly re-mapping peer.service values
+              #
+              # @default `DD_TRACE_PEER_SERVICE_MAPPING` environment variable converted to hash
+              # @return [Hash]
+              option :peer_service_mapping do |o|
+                o.default do
+                  env_to_hash(
+                    Tracing::Configuration::Ext::SpanAttributeSchema::ENV_PEER_SERVICE_MAPPING,
+                    {},
+                    comma_separated_only: true
+                  )
+                end
+              end
+
+              # Global service name behavior
+              settings :global_default_service_name do
+                # Overrides default service name to global service name
+                #
+                # Allows for usage of v1 service name changes without
+                # being forced to update schema versions
+                #
+                # @default `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` environment variable, otherwise `false`
+                # @return [Boolean]
+                option :enabled do |o|
+                  o.default { env_to_bool(Tracing::Configuration::Ext::SpanAttributeSchema::ENV_GLOBAL_DEFAULT_SERVICE_NAME_ENABLED, false) }
+                end
```

Comment:
```suggestion
                option :enabled do |o|
                  o.env Tracing::Configuration::Ext::SpanAttributeSchema::ENV_GLOBAL_DEFAULT_SERVICE_NAME_ENABLED
                  o.default false
                end
```

After rebasing master, we can use the latest configuration options